src/webview/MessageListWeb.js

```diff
@@ -36,8 +60,10 @@ export default class MessageListWeb extends Component<Props> {
 
   handleMessage = (event: { nativeEvent: { data: string } }) => {
     const eventData: MessageListEvent = JSON.parse(event.nativeEvent.data);
-    if (eventData.type === 'ready') {
-      this.isReady = true;
+    if (eventData.type === 'ping') {
```

Comment:
Looking at all this renaming, I am not sure my suggestion for 'hello' is optimal.
I think keeping the name 'ready' is superior both to 'hello' and 'ping' as it reveals the purpose for the event.